sabre1041/managing-ocp-install-beyond
# Add route53 deletion to terminate playbook

To test, deploy ideally at least 2 clusters with the following variables set:

```
student_count: 2
tower_config: true
tower_workflow_job_deploy_launch: true
tower_workflow_job_scaleup_launch: true
tower_project_provision_and_configure_url: "https://github.com/vvaldez/summit-2017-ocp-operator.git"
tower_project_provision_and_configure_branch: "route53-cleanup"
```

Once deployed (nearly an hour later) login to Tower:
- Check in AWS console under route53 DNS and verify your entries for your clusters are there (currently there seems to be a bug in which only the first tower has a route53 entry)
- Check EC2 console to verify instances are there
- Click on Inventory, then OpenShift and verify all 4 hosts are there
- Verify OpenShift console works
- Click on Job Templates
- Click "Terminate-OCP" (optionally first edit and increase verbosity if desired)
- Once complete, verify hosts are removed from Tower Inventory (Inventory -> OpenShift)
- Verify tag groups remain (Inventory -> OpenShift -> OSEv3 -> nodes (and -> masters)
- Verify route53 was removed from AWS
- Verify OCP instances are removed from AWS

Remove tower:
- Click on Job Tempaltes
- Click on "Terminate-All"
- Tower will stop responding, check AWS for route53 and instance cleanup

On second cluster:
- Under Job Templates run "Terminate-All"
